board-games/test/game-night-test.js

```diff
@@ -0,0 +1,129 @@
+var assert = require('chai').assert;
+var Player = require('../src/player.js');
+var Game = require('../src/board-game.js');
+var GameNight = require('../src/game-night.js');
+
+describe ('GameNight', function() {
+  it.skip('should have a host', function() {
+    var host = new Player({name: 'Elizabeth'});
+    var gameNight = new GameNight(host);
+
+    assert.isFunction(GameNight);
+    assert.equal(gameNight.host.name, 'Elizabeth');
+  });
+
+  it.skip('should be able to invite other gamers', function() {
+    var host = new Player({name: 'Elizabeth'});
+    var player1 = new Player({name: 'Phil'});
+    var player2 = new Player({name: 'Ryan'});
+    var gameNight = new GameNight(host);
+
+    gameNight.invite([player1, player2]);
+
+    assert.equal(gameNight.players.length, 3);
```

Comment:
Super tricky that you have to add the host as a player too!

Reply:
Sure! I added a line in this same it block.